### PR TITLE
lectia-1: Create styleguide layout

### PR DESCRIPTION
**Pull request by alexcracea**

### CODE REVIEW

**Comment by alexcracea** [comment]
Hey guys please review

**Comment by alexcracea** [comment]
@7ewen please check out

File: index.html [hunk 1]
```diff
@@ -6,8 +6,32 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>CSS Advanced</title>
   <link rel="stylesheet" href="css/main.css">
+  <link rel="stylesheet" href="css/styleguide.css">
 </head>
 <body>
-  hellow css advanced
+  <header class="header"></header>
```
Comment (alexcracea) [inline index.html:12]: please change to `header-area` and change styles for it

Comment (7ewen) [reply index.html:12]: @alexcracea agree 👍

**Review by Iul4ik — APPROVED** [review]

**Comment by VasileRatoi** [comment]
looks nice, did not tested, approved

File: index.html (hunk 1 above)
Comment (Iul4ik) [reply index.html:12]: nu este respectata tabulatia

**Review by VasileRatoi — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -6,8 +6,32 @@
...
+      </ul>
+    </aside>
+    <content class="content-area">
+      <div class="section" id="background-box-1">
+        <h2 class="section__title">Background Box 1</h2>
+        <div class="section__content">tema de azi</div>
+      </div>
+      <div class="section" id="background-box-2">
+        <h2 class="section__title">Background Box 2</h2>
+        <div class="section__content">tema de azi</div>
```
Comment (OlicikaVolosciuc) [inline index.html:28]: please change 'tema de azi'

File: scss/styleguide.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.main {
+  display: flex;
+  height: calc(100vh - $header-height);
+}
+
+.sidebar-area {
+  width: $sidebar-width;
+  background-color: pink;
```
Comment (GR-Viktor) [inline scss/styleguide.scss:18]: Mmmm... Pink )))

File: index.html (hunk 2 above)
Comment (alexcracea) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (alexcracea) [reply index.html:12]: all according to specs, mark as resolved

**Pull request merged**